files/en-us/web/javascript/data_structures/index.html

```diff
@@ -49,7 +49,7 @@ <h2 id="Data_and_Structure_types">Data and Structure types</h2>
  </li>
 </ul>
 
-<p>Keep in mind the only valuable purpose of <code>typeof</code> operator usage is checking the Data Type. If we wish to check any Structural Type derived from Object it is pointless to use <code>typeof</code> for that, as we will always receive <code>"object"</code>. The indeed proper way to check what sort of Object we are using is {{jsxref("Operators/instanceof", "instanceof")}} keyword. But even in that case there might be misconceptions.</p>
+<p>Keep in mind the only valuable purpose of <code>typeof</code> operator usage is checking the Data Type. If we wish to check any Structural Type derived from Object it is pointless to use <code>typeof</code> for that, as we will always receive <code>"object"</code>. The proper way to check what sort of Object we are using is the {{jsxref("Operators/instanceof", "instanceof")}} keyword indeed. But even in that case there might be misconceptions.</p>
```

Comment:
```suggestion
<p>Keep in mind the only valuable purpose of <code>typeof</code> operator usage is checking the Data Type. If we wish to check any Structural Type derived from Object it is pointless to use <code>typeof</code> for that, as we will always receive <code>"object"</code>. The proper way to check what sort of Object we are using is the {{jsxref("Operators/instanceof", "instanceof")}} keyword. But even in that case there might be misconceptions.</p>
```

Thanks @RaisinTen - I don't think we need the word 'indeed' at all here :)

Reply:
@Rumyra thanks for reviewing. I committed your suggestion. PTAL.